### PR TITLE
Use auth service instead of kahuna Panda

### PR DESCRIPTION
**Pull request by theefer**
Switch to use the new `auth` service instead of `kahuna`'s Panda endpoints to (re)auth in the kahuna web app.

Release order:
1. auth
2. media-api
3. kahuna

### CODE REVIEW

**Comment by theefer** [comment]
We'll want to do some further testing in TEST but this is ready to go as CNAMES are ready.

File: kahuna/conf/routes [hunk 1]
```diff
@@ -17,6 +17,8 @@ GET     /login                      controllers.Panda.doLogin(redirectUri: Optio
 GET     /oauthCallback              controllers.Panda.oauthCallback
 GET     /logout                     controllers.Panda.logout
 GET     /session                    controllers.Panda.session
+# Empty page to return to the same domain as the app
+GET     /ok                         controllers.Application.ok
```
Comment (kenoir) [inline kahuna/conf/routes]: Can we use the `/management/healthcheck` uri for this purpose?

Comment (theefer) [inline kahuna/conf/routes]: I would rather not.

Healthcheck is for healthcheck purposes (ELB, Pingdom, other monitoring tools). It's a system-facing endpoint.

The "OK" page is a user-facing landing page.

**Comment by kenoir** [comment]
:+1: Other than ^

**Pull request merged**